app/src/main/res/layout/fragment_student_details_input.xml

```diff
@@ -89,6 +89,7 @@
 
     <Button
         android:id="@+id/student_join_class_btn"
+        style="@style/Widget.Material3.Button.TonalButton"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_margin="20dp"
```

Comment:
Should use `MaterialButton` class instead of `Button` which belongs to `AppCompat`